third-party/realdds/include/realdds/topics/notification/notification-msg.h

```diff
@@ -30,28 +34,30 @@ namespace topics {
 
     
 namespace raw {
-namespace device {
 class notificationPubSubType;
-class notification;
-}  // namespace device
 }  // namespace raw
 
 
-namespace device {
-
-    
 class notification
 {
 public:
-    using type = raw::device::notificationPubSubType;
+    using type = raw::notificationPubSubType;
+
+    enum class data_type
```

Comment:
I think `data_formatting` will be a better name

Reply:
It's the type of the data... `data_format` is OK I guess but why?

Reply:
Sounds better to me :-)
Data type might mean other stuff as well. Format just means "the way in which something is arranged or set out."
Also when looking JSON and CBOR, wikipedia specifies they are data formats

Reply:
Changed.